R/g.report.part5.R

```diff
@@ -1,6 +1,7 @@
 g.report.part5 = function(metadatadir = c(), f0 = c(), f1 = c(), loglocation = c(),
                           includenightcrit = c(), includedaycrit = c(), data_cleaning_file = c(),
-                          includedaycrit.part5 = 2/3,
+                          includedaycrit.part5 = 2/3, segmentWEARcrit.part5 = 0.5,
+                          segmentDAYSPTcrit.part5 = c(0,0), 
```

Comment:
Please move segmentDAYSPTcrit.part5 and segmentWEARcrit.part5 to params_cleaning. Further, their values should be set in load_params() otherwise we risk that parameters are not consistent between parts of the documentation.

Reply:
These arguments are indeed set in `load_params()`. Here it is the definition of `g.report.part5` when called within `GGIR()`:

![image](https://github.com/wadpac/GGIR/assets/28734336/eb51d11d-074d-4cd2-8611-670cf8385e0c)

I did it this way to be consistent with the other arguments in the function, for example `include[day/night]crit` arguments are set in the same way.

Reply:
Thanks for clarifying. We now have more than 6 arguments come from the `params_cleaning` object. It may be tidier to replace all of them by a single `params_cleaning` argument to function `g.report.part5`. Next, we can use the values inside the function as in `params_cleaning[["segmentDAYSPTcrit.part5"]]`. Advantage: 
- shortens documentation for g.report.part5().
- simplifies code as less arguments have to be passed from function to function.
- ensures that we always use an argument value that has been checked by check_params() rather than a local copy.

Reply:
@jhmigueles I just made one minor commit in which I rephrase a sentence in the documentation? I think this is the last action point before I can approve the PR, shall I do it or are you currently already working on this?